Tasks/Go/main.ts

```diff
@@ -0,0 +1,54 @@
+import tl = require("vsts-task-lib/task");
+import tr = require("vsts-task-lib/toolrunner");
+import * as path from 'path';
+
+try {
+    tl.setResourcePath(path.join(__dirname, "task.json"));
+} catch (error) {
+    tl.setResult(tl.TaskResult.Failed, error);
+}
+
+export class goExe {
+    private command: string = "";
+    private argument: string = "";
+    private workingDir: string = "";
+    private failOnStdErr: boolean = false;
+
+    constructor() {
+        this.command = tl.getInput("command", true).trim();
+        if (this.command == "custom") {
+            this.command = tl.getInput("customCommand", true).trim();
+        }
+        this.argument = tl.getInput("arguments", false);
+        this.workingDir = tl.getInput("workingDirectory", false);
+        if (this.workingDir == null) {
+            this.workingDir = tl.getVariable("System.DefaultWorkingDirectory");
```

Comment:
I think system.defaultworkingdir is default in tool runner if workingDir is null